### DISCUSSION TITLE
Allow inspecting serverless function files

### DISCUSSION THREAD

**Discussion by franciscorubin** [vercel/vercel#5012] [Help]
I am having trouble debugging my serverless functions, which fail for being too big:

```
The Serverless Function "index" is 164.95mb which exceeds the maximum size limit of 50mb.
```

I think there should be a way to inspect your functions, including the file bundle that gets uploaded to them. Right now the only way I've found to debug this is using trial and error blindly, generating multiple deploys.

**Comment by styfle** [comment]
Thanks for the suggestion!

In the meantime, you can use https://packagephobia.com to identify large dependencies.
